src/iuse_actor.cpp

```diff
@@ -1346,12 +1325,13 @@ long inscribe_actor::use( player *p, item *it, bool t, const tripoint& ) const
         return it->type->charges_to_use();
     }
 
-    return 0;
+    return cost >= 0 ? cost : it->ammo_required();
```

Comment:
This is the wrong place for that change. Here it should always return 0. Change a few lines above (`return charges_to_use`) and you probably also need to handle the result of `iuse::handle_ground_graffiti` as it currently also returns `charges_to_use`.